src/routes/api/draftarticles.js

```diff
@@ -0,0 +1,8 @@
+// import express from 'express';
```

Comment:
Why those useless comments and not JDOC!?

Reply:
Resolved